### DISCUSSION TITLE
Tabs Vertical

### DISCUSSION THREAD

**Discussion by bensonarafat** [tailwindlabs/headlessui#2149] [Help] [answered]
I am adding the vertical props to tabs but it's not working for me. 
I am using Vue 

Thank You.

**Comment by RobinMalfait** [comment, marked as answer]
Hey!

The vertical prop changes the way keyboard shortcuts are handled (ArrowDown vs ArrowRight) and changes the `aria-orientation` for example. You still have to do the styling yourself to make it look vertical.

**Reply by bensonarafat** [reply]
Oh. Okay! 
Got it

**Reply by juniorEdris** [reply]
just realized after reading this comment
